namespacecost.go

```diff
@@ -0,0 +1,43 @@
+package main
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"net/url"
+
+	"github.com/gorilla/mux"
+)
+
+func (s *server) GetCost(w http.ResponseWriter, r *http.Request) {
+	vars := mux.Vars(r)
+	namespace := vars["namespace"]
+
+	log.Printf("loading namespace costs for %q", namespace)
+
+	params := url.Values{}
+	params.Add("namespace", namespace)
+
+	resp, err := http.Get("http://localhost:9090/model/aggregatedCostModel?window=1d&aggregation=namespace&" + params.Encode())
```

Comment:
* URL needs to be an application parameter. This will only work locally.
* Use `url.Values` for either all query parameters, or none of them (via `QueryEscape`).